test/repo/Makefile

```diff
@@ -81,3 +82,15 @@ compare:
 	@ for html in $$(cd $(REPORT_ROOT2) && find . -name html); do \
 	  diff -rq $(REPORT_ROOT1)/$$html $(REPORT_ROOT2)/$$html; \
 	done
+
+summarize:
+	$(RM) -r $(REPORT_ROOT)
+	cp $(PACKAGE_ROOT)/src/cbmc_starter_kit/template-for-repository/proofs/run-cbmc-proofs.py $(REPO_ROOT)/$(REPO_PROOF_ROOT)
+	cd $(REPO_ROOT)/$(REPO_PROOF_ROOT) && \
```

Comment:
I suggest to `cd` on each line, don't do `&& \`. Using the line continuation means that this entire command will be executed in a subshell, and it's difficult to know which command failed if any.

Reply:
Done. I just pushed a separate commit that does that.